docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md

```diff
@@ -193,7 +193,7 @@ Pageview tracking is used to record views of web pages.
 
 Currently, recording a pageview involves recording an event where `e=pv`. All the fields associated with web events can be tracked. There are no other pageview specific fields:
 
-```
+```text
```

Comment:
should this really be a table instead?

Reply:
Potentially. I think this whole page could be a bit prettier to be honest! I'll make a follow up which tidies up the tracker protocol page.